lib/ansible/modules/messaging/rabbitmq_policy.py

```diff
@@ -70,6 +70,13 @@
       - Erlang node name of the rabbit we wish to configure.
     required: false
     default: rabbit
+  extra_ctl_paths:
+    description:
+      - List of alternative paths to look for rabbitmqctl in
+      - Only needed when running RabbitMQ as user other than root / rabbitmq
+    required: false
+    default: ()
+    version_added: "2.4"
```

Comment:
This isn't common because it's not used in as many modules as it is used in. The following files do not use it:

- rabbitmq_binding.py
- rabbitmq_exchange.py
- rabbitmq_plugin.py
- rabbitmq_queue.py

The following _do_ use it

- rabbitmq_parameter.py
- rabbitmq_policy.py
- rabbitmq_user.py
- rabbitmq_vhost.py

It's not an appropriate parameter in files that do not use the rabbitmqctl binary, so I don't believe that they're common.